support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -35,6 +37,25 @@ interface PaymentTsAndCsProps {
 	amountIsAboveThreshold: boolean;
 }
 
+const manageMyAccount = (
```

Comment:
Do we need to check whether they are logged in? will the link work if they are not a registered user?